### PR TITLE
Update README build deps info for Linux

### PR DESCRIPTION
**Pull request by sombraguerrero**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,8 +43,29 @@ You must also have Java installed, and available on your PATH as `java`.
...
+* Java (OpenJDK 11 will suffice)
+* Libgtk-3-dev
+* Libssl-dev
+* Libasound2-dev
+* \*gobject\*
+* libxcb\*
+* cmake
+* g++
+
+<h3>Fedora</h3>
```
Comment (relrelb) [inline README.md:58]: Any reason for not using Markdown's `####`?

File: README.md [hunk 2]
```diff
@@ -43,8 +43,29 @@ You must also have Java installed, and available on your PATH as `java`.
 
 ### Desktop
 
-If you are building for a Linux platform, make sure that the GTK 3 development packages are
-installed on your system. (Ubuntu: `libgtk-3-dev`, Fedora: `gtk3-devel`)
+If you are building for a Linux platform, the following are typical dependencies:
+<h3>Ubuntu</h3>
+
+* Java (OpenJDK 11 will suffice)
+* Libgtk-3-dev
```
Comment (relrelb) [inline README.md:50]: Let's keep the names lower-cased, as they appear in command-line.

File: README.md [hunk 3]
```diff
@@ -43,8 +43,29 @@ You must also have Java installed, and available on your PATH as `java`.
...
+* Libssl-dev
+* Libasound2-dev
+* \*gobject\*
```
Comment (relrelb) [inline README.md:53]: Is there any special meaning for the surrounding `*`?

Comment (sombraguerrero) [reply README.md:53]: There are enough packages within that particular group of dependencies that I feel it's just more expedient to wildcard the entire term.

Comment (adrian17) [reply README.md:53]: Probably a wildcard, as there isn't just one package called gobject. That said, we definitely shouldn't recommend to install them manually, right? They aren't explicitly needed on Actions workers after all.

Comment (sombraguerrero) [reply README.md:53]: Perhaps not, but my compilations have always errored without them.

**Review by relrelb — APPROVED** [review]

**Comment by relrelb** [comment]
Thank you!

**Comment by sombraguerrero** [comment]
> Thank you!

You're welcome, @relrelb

**Pull request merged**